### ISSUE TITLE
Debug NIST issues on the latest Release

### ISSUE THREAD

**Issue by mscaldas2012** [CDCgov/data-exchange-hl7#453]
Currently getting a NPE when tryng to load profiles with the latest NIST release (1.5.5)

NIST team created a 1.5.6, but still having issues.

**Comment by mscaldas2012** [comment]
Have a call with Hossam (NIST Team) Today - 1/5/23 to see if we can find environmental issues since they claim it's working on their environment.

**Comment by mscaldas2012** [comment]
NIST has released version 1.5.6-SNAPSHOT. We narrow down the issue to the NIST libraries not being able to load certain schema files - ConformanceContext.xsd and its dependencies, Extension.xsd and Commons.xsd.

As a stop gap, I've downloaded those schemas from source code from NIST and added them directly under resources folder on our project. That seems to mitigate the problem while NIST folks further investigate why that's an issue on certain environments.

No windows environment runs properly withouth the local XSD. Some macs run ok, some don.t - we tried two different JDKs - openJDK and aws correto JDK (all v.11)
We also ran sucessfully on linux under windows subssytem on a cdc machine.
